components/script/dom/element.rs

```diff
@@ -137,6 +137,8 @@ pub struct Element {
     /// when it has exclusive access to the element.
     #[ignore_heap_size_of = "bitflags defined in rust-selectors"]
     selector_flags: Cell<ElementSelectorFlags>,
+    #[ignore_heap_size_of = "RwLock"]
+    cached_lang: RwLock<Option<Atom>>,
```

Comment:
To be precise, not only if the lang attribute is cached, but also one of the other conditions that could make lang change its value.